doc/release-notes.md

```diff
@@ -4,3 +4,14 @@ release-notes at release time)
 Notable changes
 ===============
 
+Option handling
+---------------
+
+- A new `-preferredtxversion` argument has been added to allow the node
+  to preferentially create transactions of a specified version, if a
+  transaction does not contain components that necessitate creation with a
+  specific version. For example, setting `-preferredtxversion=4` will cause
+  the node to create V4 transactions in any case where the transaction does not
+  contain Orchard components. This can be helpful if recipients of transactions
+  are likely to be using legacy wallets that have not yet been upgraded to
+  support parsing V5 transactions.
```

Comment:
The wallets don't have to fully support V5 transaction creation, but they *do* have to be able to parse v5 transactions. A wallet won't generate a UA containing an Orchard receiver if it can't spend those funds, so all that's required is the ability to parse.